src/pages/ja/install/auto.mdx

```diff
@@ -1,67 +1,84 @@
 ---
-title: 自動CLIでAstroをインストール
+title: Astroを自動CLIでインストール
 description: npm、pnpm、またはYarnを使って、create-astro CLIツールによりAstroをインストールする方法です。
 layout: ~/layouts/MainLayout.astro
 i18nReady: true
 ---
 import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 Astroをインストールする準備はできましたか？自動または手動セットアップガイドにしたがって開始してください。
 
 #### 前提条件
 
-- **Node.js** - `14.15.0`、`v16.0.0`、またはそれ以上。
+- **Node.js** - `14.18.0`、`v16.12.0`、またはそれ以上。
 - **テキストエディタ** - [VS Code](https://code.visualstudio.com/)と[公式Astro拡張機能](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode)を推奨します。
 - **ターミナル** - Astroは、コマンドラインインターフェイス（CLI）からアクセスします。
 
 <InstallGuideTabGroup />
 
 #### インストール
 
-`create-astro` は、新しいAstroプロジェクトをゼロから始めるもっとも速い方法です。
+`create-astro` は、新しいAstroプロジェクトをゼロから始めるもっとも速い方法です。新しいAstroプロジェクトをセットアップするためのすべてのステップを案内してくれます。いくつかの異なるスターターテンプレートから選んだり、 `--template` 引数を使用して独自のテンプレートを使用できます。
 
 :::tip[オンラインプレビュー]
-ブラウザでAstroを試してみませんか？[Astro.new](https://astro.new/)では、スターターテンプレートを利用し、ブラウザから離れることなく、新しいAstroプロジェクトを立ち上げられます。
+ブラウザでAstroを試したいですか？[Astro.new](https://astro.new/)では、スターターテンプレートを利用し、ブラウザから離れることなく、新しいAstroプロジェクトを立ち上げられます。
 :::
 
 ## 1. セットアップウィザードを実行する
 
 ターミナルで以下のコマンドを実行すると、便利なインストールウィザード、`create-Astro`が起動します。
 
-```shell
-# npm
-npm create astro@latest
-
-# yarn
-yarn create astro
-
-# pnpm
-pnpm create astro@latest
-```
-
-`create-astro`ウィザードは、新しいAstroプロジェクトのセットアップの全ステップを案内します。このウィザードはどのディレクトリでも実行でき、始める前にプロジェクト用の新しい空のディレクトリを作成する必要はありません。新しいプロジェクト用の空のディレクトリがまだなければ、ウィザードが自動的に作成します。
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # npmで新しいプロジェクトを作成する
+  npm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # pnpmで新しいプロジェクトを作成する
+  pnpm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # yarnで新しいプロジェクトを作成する
+  yarn create astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+`create-astro`はマシンの任意の場所で実行でき、始める前にプロジェクト用の新しい空のディレクトリを作成する必要はありません。新しいプロジェクト用の空のディレクトリがまだなければ、ウィザードが自動的に作成します。
 
 すべてがうまくいけば、「Ready for liftoff!」というメッセージの後に、いくつかの推奨する「Next steps」が表示されるはずです。新しいプロジェクトディレクトリに`cd`で移動し、Astroの使用を開始します。
 
 もし`create-astro`ウィザードで`npm install`のステップをスキップした場合は、続行する前に依存関係をインストールしてください。
 
-
 ## 2. Astroをスタートする ✨
 
-Astroには、プロジェクト開発に必要なものをすべて備えた開発サーバーが内蔵されています。`Astro dev`コマンドを実行すると、ローカルの開発サーバーが起動し、新しいウェブサイトが実際に動作しているところをいち早く見られます。
+Astroには、プロジェクト開発に必要なものをすべて備えた開発サーバーが内蔵されています。`astro dev`コマンドを実行すると、ローカルの開発サーバーが起動し、新しいウェブサイトが実際に動作しているところを初めて見られるようになります。
```

Comment:
まず、`Astro dev` を `astro dev` へと修正しました。

さらに、第二文の原文は

> The `astro dev` command will start the local development server so that you can see your new website in action for the very first time.

となっており、for the very first time の訳が「いち早く」となっていたのですが、ここは単純に「初めて」という意味であると思われるため、「初めて」に変更しています。ただ、そのまま「初めて見られます」だとどうもカタイ感じがし、「ここまでの過程を経てとうとうご対面できます！」という雰囲気を醸し出したほうがいい気がしたため、「初めて見られるようになります」と少し意訳しています。